tests/Benchmarks/System.IO.Pipelines/E2E.cs

```diff
@@ -8,11 +8,9 @@
 using System.Text.Formatting;
 using System.Text.JsonLab;
 using BenchmarkDotNet.Attributes;
-using BenchmarkDotNet.Engines;
 
 namespace System.IO.Pipelines.Benchmarks
 {
-    [SimpleJob(RunStrategy.Monitoring)] // the setting for benchmarks with not-steady state
```

Comment:
What changed that allows us to remove this attribute?

Reply:
I made BDN smarter ;) Now when the benchmark is very time consuming it will run it once per iteration, not 16 times as it used to.